psdk_wrapper/include/psdk_wrapper/psdk_wrapper_utils.hpp

```diff
@@ -22,6 +22,7 @@
 #include <tf2/LinearMath/Matrix3x3.h>
 #include <tf2/LinearMath/Quaternion.h>
 
+#include <map>
```

Comment:
is this `#include <map>` being used?

Reply:
Indeed not necessary, removed in last commit